userguide/network.rst

```diff
@@ -737,6 +737,18 @@ More information about this command can be found at
 `systat(1) <https://www.freebsd.org/cgi/man.cgi?query=systat>`__.
 
 
+.. _Network Summary:
+
+Network Summary
+---------------
+
+:menuselection:`Network --> Network Summary`
+shows a quick summary of the addressing information of every
+configured interface. For each interface name, the configured IPv4 and
+IPv6 addresses, DNS servers, and default gateway are displayed. It
```

Comment:
Is "default gateway" supposed to be the "Default Routes" I see in the page, or is "Default Routes" another object?

Reply:
Default gateway and default router are the same thing.  Default routes are actual routes, but I don't think that term was used.  IP addresses and default gateway are basic information, DNS nameservers are secondary.